diff --git a/pyproject.toml b/pyproject.toml
index d7d4995d..5f4b99c7 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -40,7 +40,7 @@ dependencies = [
     "numpy >= 1.20, < 2.0",
     "scipy >= 1.4",  #NB: for cython3 SciPy>=1.11.2, see !159#note_1619762810.
     "matplotlib < 3.9",
-    "networkx >= 2.4",
+    "networkx < 3.3",
     "sly >= 0.4",
     "click >= 7.1",
     "click-default-group >= 1.2.2",
