diff --git i/src/finesse/utilities/text.py w/src/finesse/utilities/text.py
index aa574757d..da61bcf0f 100644
--- i/src/finesse/utilities/text.py
+++ w/src/finesse/utilities/text.py
@@ -5,7 +5,6 @@ from collections.abc import Iterable
 import spellchecker
 from quantiphy import Quantity as _Quantity
 from colorama import init, Cursor
-from IPython.display import clear_output
 from finesse.env import is_interactive as is_jupyter
 from typing import List
 
@@ -29,6 +28,7 @@ def rewrite_lines(lines: List[str], clear: bool = True):
         Clear the output before writing new lines. Defaults to `True`.
     """
     if is_jupyter():
+    	   from IPython.display import clear_output
         # Jupyter: Use clear_output
         if clear:
             clear_output(wait=True)
