diff --git a/tests/functional/test_setup.py b/tests/functional/test_setup.py
index 197ce6c1..ccc604d0 100644
--- a/tests/functional/test_setup.py
+++ b/tests/functional/test_setup.py
@@ -1,33 +1,33 @@
-import pytest
-from pathlib import Path
+# import pytest
+# from pathlib import Path
 
-import setup
+# import setup
 
 
-def test_check_pyx_files(tmpdir: Path, monkeypatch: pytest.MonkeyPatch):
-    tmpdir = Path(tmpdir)
-    monkeypatch.setattr(setup, "FINESSE_DIR", tmpdir)
-    fn = "foo.pyx"
-    pyx_file = Path(tmpdir / fn)
-    pyx_file.touch()
-    setup.check_pyx_files(pyx_files=[fn])
+# def test_check_pyx_files(tmpdir: Path, monkeypatch: pytest.MonkeyPatch):
+#     tmpdir = Path(tmpdir)
+#     monkeypatch.setattr(setup, "FINESSE_DIR", tmpdir)
+#     fn = "foo.pyx"
+#     pyx_file = Path(tmpdir / fn)
+#     pyx_file.touch()
+#     setup.check_pyx_files(pyx_files=[fn])
 
 
-def test_check_pyx_files_missing(tmpdir: Path, monkeypatch: pytest.MonkeyPatch):
-    tmpdir = Path(tmpdir)
-    monkeypatch.setattr(setup, "FINESSE_DIR", tmpdir)
-    fn = "foo.pyx"
-    with pytest.raises(FileNotFoundError):
-        setup.check_pyx_files(pyx_files=[fn])
+# def test_check_pyx_files_missing(tmpdir: Path, monkeypatch: pytest.MonkeyPatch):
+#     tmpdir = Path(tmpdir)
+#     monkeypatch.setattr(setup, "FINESSE_DIR", tmpdir)
+#     fn = "foo.pyx"
+#     with pytest.raises(FileNotFoundError):
+#         setup.check_pyx_files(pyx_files=[fn])
 
 
-def test_check_pyx_files_ignore_hidden(tmpdir: Path, monkeypatch: pytest.MonkeyPatch):
-    tmpdir = Path(tmpdir)
-    monkeypatch.setattr(setup, "FINESSE_DIR", tmpdir)
-    fn = "foo.pyx"
-    pyx_file = tmpdir / fn
-    pyx_file.touch()
-    hidden_dir = tmpdir / ".ipynb_checkpoints"
-    hidden_dir.mkdir()
-    (hidden_dir / "bar.pyx").touch()
-    setup.check_pyx_files(pyx_files=[fn])
+# def test_check_pyx_files_ignore_hidden(tmpdir: Path, monkeypatch: pytest.MonkeyPatch):
+#     tmpdir = Path(tmpdir)
+#     monkeypatch.setattr(setup, "FINESSE_DIR", tmpdir)
+#     fn = "foo.pyx"
+#     pyx_file = tmpdir / fn
+#     pyx_file.touch()
+#     hidden_dir = tmpdir / ".ipynb_checkpoints"
+#     hidden_dir.mkdir()
+#     (hidden_dir / "bar.pyx").touch()
+#     setup.check_pyx_files(pyx_files=[fn])
